### DISCUSSION TITLE
Is it not possible to host through Intel igpu on Windows?

### DISCUSSION THREAD

**Discussion by ytoaa** [LizardByte/community#89] [Sunshine] [answered]
### Describe the Bug

It was encoded via vaapi in a Linux environment, but is there no way to do it in Windows?

### Operating System Version

windows 11

### Architecture

64bit

### Sunshine Version

0.14.1

### GPU Type

igpu(intel)

### GPU Model

intel uhd

### GPU Driver/Mesa Version

nope

**Comment by LC95** [comment]
same question

[a comment by an automated account is not shown]
